languages/elixir/exercises/concept/anonymous-functions/.meta/example.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Secrets do
+
+  def secret_add(secret) do
+    fn x -> x + secret end
+  end
+
+  def secret_subtract(secret) do
+    fn x -> x - secret end
+  end
+
+  def secret_multiply(secret) do
+    fn a -> a * secret end
+  end
+
+  def secret_divide(secret) do
+    fn x -> div(x, secret) end
+  end
+
+  def secret_combine(f, g) do
+    fn x ->
+      x |> f.() |> g.()
```

Comment:
The [example file](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#example-implementation-file) should only use language features that are know to the student. Is the pipe operator already know at this point?